src/ddmd/aggregate.h

```diff
@@ -230,23 +230,6 @@ struct BaseClass
     void copyBaseInterfaces(BaseClasses *);
 };
 
-struct ClassFlags
-{
-    typedef unsigned Type;
```

Comment:
Will this negatively affect the C++ interface?  cc @ibuclaw

Reply:
@kinke @JohanEngelen would this be a problem for LDC?

Reply:
Yes it would indeed: https://github.com/ldc-developers/ldc/blob/d1b30d627c5bbb58ff306aa924805871dbda806d/gen/classes.cpp#L491

Reply:
Ditto gdc.

To keep everyone synchronised, it should remain in the frontend.